common/changes/office-ui-fabric-react/contextualmenu-section_2019-01-17-03-35.json

```
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Fixes #7686",
```

This comment is meant to serve as release notes. I would suggest something like this:
```suggestion
      "comment": "ContextualMenu: fixes issue #7686 when items of type `section` are given the same key and later are rendered as list items causing a React warning. Change the example having this issue.",
```